apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs

```diff
@@ -182,9 +182,29 @@ function writeNodeRow(nodeID, nodeName, nodeStatus, buttonType, addlClass) {
           writeNodeRow(row.nodeID, row.displayName, row.status, buttonType, row.classAttr);
         });
 
+        var buttons = [];
+
         if (!controller.hasOutgoingRequest() && levelConfig.showAddRouterButton) {
           var buttonText = levelConfig.broadcastMode ? i18n.addRoom() : i18n.addRouter();
-          writeEmptyRow(buttonMarkup(buttonText, 'netsim-lobby-add-router', ['secondary']));
+          buttons.push(buttonMarkup(
+              buttonText,
+              'netsim-lobby-add-router',
+              ['secondary']));
```

Comment:
Nit: I would personally put the parens on the next line, indented the same as line 189 to make it easier to match open/close. This might just be personal preference though, and you're welcome to do whatever.